tests/TestCase/Error/Middleware/ErrorHandlerMiddlewareTest.php

```diff
@@ -143,6 +145,55 @@ public function testHandleException()
         $this->assertStringContainsString('was not found', '' . $result->getBody());
     }
 
+    /**
+     * Test creating a redirect response
+     *
+     * @return void
+     */
+    public function testHandleRedirectException()
+    {
+        $request = ServerRequestFactory::fromGlobals();
+        $middleware = new ErrorHandlerMiddleware();
+        $handler = new TestRequestHandler(function () {
+            throw new RedirectException('http://example.org/login');
```

Comment:
For security purposes, shouldn't we only use internal (relative) URLs?
https://github.com/cakephp/cakephp/pull/14384/files#diff-fd25dba57f71dd4309af4df2494d43e2R55
Or is that up to the user at this point?

Reply:
I think that is up to the user. The exceptions and redirect response don't modify the provided url at all.